packages/plugin-expo-app/package-lock.json

```diff
@@ -101,28 +101,6 @@
 				"to-fast-properties": "^2.0.0"
 			}
 		},
-		"@bugsnag/core": {
-			"version": "7.0.0-pre-alpha-ben.6",
-			"resolved": "https://registry.npmjs.org/@bugsnag/core/-/core-7.0.0-pre-alpha-ben.6.tgz",
-			"integrity": "sha512-NL33soHPbL+8xm/jIMPlIOIzUErEJgelDC3HOqSMNwPpGidE5768H28MUFGTF77TatRkihcZS5MNf4Ymt888Pg==",
-			"requires": {
-				"@bugsnag/cuid": "^3.0.0",
-				"@bugsnag/safe-json-stringify": "4.0.0",
-				"error-stack-parser": "^2.0.2",
-				"iserror": "0.0.2",
-				"stack-generator": "^2.0.3"
-			}
-		},
-		"@bugsnag/cuid": {
-			"version": "3.0.0",
-			"resolved": "https://registry.npmjs.org/@bugsnag/cuid/-/cuid-3.0.0.tgz",
-			"integrity": "sha512-LOt8aaBI+KvOQGneBtpuCz3YqzyEAehd1f3nC5yr9TIYW1+IzYKa2xWS4EiMz5pPOnRPHkyyS5t/wmSmN51Gjg=="
-		},
-		"@bugsnag/safe-json-stringify": {
-			"version": "4.0.0",
-			"resolved": "https://registry.npmjs.org/@bugsnag/safe-json-stringify/-/safe-json-stringify-4.0.0.tgz",
-			"integrity": "sha512-eb0yG+PNTdpRXIhfEq4IFi2wBfpE1DjrAlkyehinu9RxE1PKteqJioLmfQjvkG3UviGu/k5pcamYMxNuCAMIoQ=="
-		},
```

Comment:
These lines were removed by lerna bootstrap. I believe this change is correct as the changed packages now reflect the other packages that do not contain these entries in their package-lock.json which makes things more consistent.

Reply:
Yeah it looks like these got a little messed up and stopped linking locally to `@bugsnag/core` and instead pulling that prerelease version down from npm.

I agree these changes seem correct.